test/run_tests.py

```diff
@@ -476,6 +510,46 @@ def get_deps(test_number, tests):
 
     return list(tests_to_run_explicitly)
 
+def convert_tests_for_flatbuffers(tests, to_flatbuff, working_dir, color_enum):
+    test_base_working_dir = str(working_dir)
+    if not Path(test_base_working_dir).exists():
+        Path(test_base_working_dir).mkdir(parents=True, exist_ok=True)
+
+    for test in tests:
+        test_id = test['id']
+        if 'vw_command' not in test:
+            print("{}Skipping test {} for flatbuffers, no vw command available{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'flatbuffer' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, already a flatbuffer test{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'malformed' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, malformed input{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'input_files' not in test:
+            print("{}Skipping test {} for flatbuffers, no input files{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'dictionary' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, currently dictionaries are not supported{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'invert_hash' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, invert_hash not supported on converted files{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'audit' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, audit not supported{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'dsjson' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, dsjson coming soon{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'cats' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, continuous actions coming soon{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+
+        fb_test_converter = fb_converter.FlatbufferTest(test, working_dir)
+        fb_test_converter.to_flatbuffer(to_flatbuff, color_enum)
```

Comment:
if you want to avoid pushing color_enum around you can define it before importing runtests_flatbuffer_converter and then you can import it as usual on runtests_flatbuffer_converter.py. Another option would be to have a runtest_util.py but that might be overkill. Another option would be to move the import of runtests_flatbuffer_converter either to line 514 or 547 for it to be imported right where it is needed (I prefer this one, but its a matter of style/taste).

Reply:
`color_enum` is dependent on input arguments (it is set to NoColor or Color depending on args.no_color) and is set in `main`. I could still define it somewhere else but if I don't pass it as a var it's value will not change with the input arg preference